Pyrado/pyrado/algorithms/utils.py

```diff
@@ -269,3 +269,10 @@ def reset_rollouts(self) -> None:
     def __getattr__(self, name: str) -> Any:
         # Forward to the wrapped sampler
         return getattr(self.wrapped_sampler, name)
+
+    # Required so that pickle does not end in an endless recursion
+    def __getstate__(self):
```

Comment:
Add a docstring here.

Reply:
could move the comment above in the doc string

Reply:
Done